### PR TITLE
Performance stats repro scripts

### PR DESCRIPTION
**Pull request by howard-stearns**
Update the tests/performance scripts to match the client-side statements in the LOD and performance doc.

https://app.asana.com/0/32622044445063/47479208249713

### CODE REVIEW

**Comment by howard-stearns** [comment]
I'm marking this QA Approved because I've tested this, and it's awkward to make QA edit interface scripts to make them work with entity scripts that are in the sandbox.

**Comment by howard-stearns** [comment]
Note that this PR does not change the behavior of the tribble entity script, which will change color for each user instead of one owner. That's a separate issue (https://app.asana.com/0/32622044445063/85114103526644), which does not effect achieving the purpose of updating the scripts used in reproducing the performance test results.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 5376ba9b7d6ce4da373d7a5a57ed4cd9914bcbca available at: https://gist.github.com/76813cb874a3e3b3cc14<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6980.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6980.dmg

**Comment by howard-stearns** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Build console output for commit: 5376ba9b7d6ce4da373d7a5a57ed4cd9914bcbca available at: https://gist.github.com/76813cb874a3e3b3cc14<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6980.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6980.dmg

File: examples/entityScripts/simpleKeepAway.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    }
+
+    this.preload = function (givenEntityID) {
+        var properties = Entities.getEntityProperties(givenEntityID, ['userData']),
+            userData = properties.userData && JSON.parse(properties.userData);
+        entityID = givenEntityID;
+        if (userData) {
+            distanceRate = userData.distanceRate || distanceRate;
+            distanceAllowance = userData.distanceAllowance || distanceAllowance;
+	    distanceScale = userData.distanceScale || distanceScale;
```
Comment (sethalves) [inline examples/entityScripts/simpleKeepAway.js]: funny indent, here

File: examples/tests/performance/renderableMatrix.js [hunk 2]
```diff
@@ -11,25 +11,51 @@ var Entities, Script, print, Vec3, MyAvatar, Camera, Quat;
...
 
-var LIFETIME = 60;
+var LIFETIME = 20;
+var ROWS_X = 17;
+var ROWS_Y = 10;
+var ROWS_Z = 10;
+// Entities will be populated from this list set by the script writer for different tests.
+var TYPES_TO_USE = [ 'Box', 'Sphere'];
+switch ('primitives') { // Quickly override the above by putting here one of the following case strings.
+case 'particles':
```
Comment (sethalves) [inline examples/tests/performance/renderableMatrix.js]: I *think* our coding standard would have this entire block indented by 4 more spaces.

File: examples/tests/performance/renderableMatrix.js [hunk 3]
```diff
@@ -11,25 +11,51 @@ var Entities, Script, print, Vec3, MyAvatar, Camera, Quat;
...
+//  NOTE: to test the full rendering of the specified number of objects (as opposed to
+//  testing LOD filtering), you may want to set LOD to manual maximum visibility.
 
-var LIFETIME = 60;
+var LIFETIME = 20;
+var ROWS_X = 17;
+var ROWS_Y = 10;
+var ROWS_Z = 10;
+// Entities will be populated from this list set by the script writer for different tests.
+var TYPES_TO_USE = [ 'Box', 'Sphere'];
```
Comment (sethalves) [inline examples/tests/performance/renderableMatrix.js]: stray space

**Comment by hifi-gustavo** [comment]
Build console output for commit: 11d80cdc941ae8d335ddf076a0b89885e2445ba5 available at: https://gist.github.com/76813cb874a3e3b3cc14<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6980.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6980.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: d8069b6dbff90a205974fbacb5bfe8ec449ba082 available at: https://gist.github.com/76813cb874a3e3b3cc14<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6980.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6980.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: ae10d4343ccad3042b995a7f1fc47949d35fa014 available at: https://gist.github.com/76813cb874a3e3b3cc14<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6980.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6980.dmg

**Pull request merged**